index.html

```diff
@@ -51,29 +51,32 @@
 <p id="offlineTimer" style="color: plum"></p>
 <p id="progressbardescription" style="color: yellow">Loading Offline Progress...</p>
 <div id="offlineprogressdone"></div>
+<div id="ascensionStats">
+    <span onclick="toggleAscStatPerSecond(1)" id="AscCubeStats">
+        <img src="Pictures/WowCube.png" class="corruptionImg" alt="">
+        <div><span id="AscCubes">x</span> C<span id="unit1">/s</span></div>
+    </span>
+    <span onclick="toggleAscStatPerSecond(2)" id="AscTessStats">
+        <img src="Pictures/WowTessaract.png" class="corruptionImg" alt="">
+        <div><span id="AscTess">x</span> T<span  id="unit2">/s</span></div>
+    </span>
+    <span onclick="toggleAscStatPerSecond(3)" id="AscHyperSpan">
+        <img src="Pictures/WowHypercube.png" class="corruptionImg" alt="">
+        <div><span id="AscHyper">x</span> H<span  id="unit3">/s</span></div>
+    </span>
+    <span id="AscTimeTakenStats">
+        <img src="Pictures/questionable.png" class="corruptionImg" alt="">
+        <div><span id="AscLen">x</span></div>
+    </span>
+    <span id="AscC10CompletionsStats">
+        <img src="Pictures/Transparent%20Pics/ChallengeTen.png" class="corruptionImg" alt="">
+        <div><span id="AscC10">x</span></div>
+    </span>
+    <span id="AscTimeFactorStats">
+        <img src="Pictures/TimeAccel.png" class="corruptionImg" alt="">
+        <div><span id="AscTimeAccel">x</span></div>
+    </span>
 </div>
```

Comment:
This was a loose `</div>` in the original code. If this PR doesn't get merged, this should definitely be cleaned up.

Reply:
It was pointed out to me that this is in fact not true.